src/main/java/qna/domain/DeleteHistorys.java

```diff
@@ -0,0 +1,30 @@
+package qna.domain;
+
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public class DeleteHistorys {
```

Comment:
```suggestion
public class DeleteHistories {
```

사소한 오타가 신경쓰여서 언급해봅니다.

Reply:
복수라서 그저 s 하나만 붙이고 정작 단어가 맞는지는 신경쓰지 않았네여 ㅜㅜ